cmd/fetch/fetch.go

```diff
@@ -23,6 +25,23 @@ const (
 	cloudquery fetch`
 )
 
+// sendProviderTelemetryEvents sends all collected telemetry events from the provider fetch response.
+func sendProviderTelemetryEvents(providers registry.Providers, fr *core.FetchResponse) {
+	if fr == nil {
```

Comment:
Since the only place this is called (L64) has a `if result != nil` block before it, you can move the call one line up to L63 and get rid of this `nil` check.

Reply:
Updated as suggested.